sdk/go/dagger.gen.go

```diff
@@ -8115,57 +8115,99 @@ func (CacheSharingMode) IsEnum() {}
 
 const (
 	// Shares the cache volume amongst many build pipelines, but will serialize the writes
-	Locked CacheSharingMode = "LOCKED"
+	CacheSharingModeLocked CacheSharingMode = "LOCKED"
+
+	// Shares the cache volume amongst many build pipelines, but will serialize the writes
+	// Deprecated: use CacheSharingModeLocked instead
+	Locked CacheSharingMode = CacheSharingModeLocked
+
+	// Keeps a cache volume for a single build pipeline
+	CacheSharingModePrivate CacheSharingMode = "PRIVATE"
 
 	// Keeps a cache volume for a single build pipeline
-	Private CacheSharingMode = "PRIVATE"
+	// Deprecated: use CacheSharingModePrivate instead
+	Private CacheSharingMode = CacheSharingModePrivate
+
+	// Shares the cache volume amongst many build pipelines
+	CacheSharingModeShared CacheSharingMode = "SHARED"
 
 	// Shares the cache volume amongst many build pipelines
-	Shared CacheSharingMode = "SHARED"
+	// Deprecated: use CacheSharingModeShared instead
+	Shared CacheSharingMode = CacheSharingModeShared
 )
 
 type ImageLayerCompression string
 
 func (ImageLayerCompression) IsEnum() {}
 
 const (
-	Estargz ImageLayerCompression = "EStarGZ"
+	ImageLayerCompressionEstargz ImageLayerCompression = "EStarGZ"
 
-	Gzip ImageLayerCompression = "Gzip"
+	// Deprecated: use ImageLayerCompressionEstargz instead
+	Estargz ImageLayerCompression = ImageLayerCompressionEstargz
 
-	Uncompressed ImageLayerCompression = "Uncompressed"
+	ImageLayerCompressionGzip ImageLayerCompression = "Gzip"
 
-	Zstd ImageLayerCompression = "Zstd"
+	// Deprecated: use ImageLayerCompressionGzip instead
+	Gzip ImageLayerCompression = ImageLayerCompressionGzip
+
+	ImageLayerCompressionUncompressed ImageLayerCompression = "Uncompressed"
+
+	// Deprecated: use ImageLayerCompressionUncompressed instead
+	Uncompressed ImageLayerCompression = ImageLayerCompressionUncompressed
+
+	ImageLayerCompressionZstd ImageLayerCompression = "Zstd"
+
+	// Deprecated: use ImageLayerCompressionZstd instead
+	Zstd ImageLayerCompression = ImageLayerCompressionZstd
 )
 
 type ImageMediaTypes string
 
 func (ImageMediaTypes) IsEnum() {}
 
 const (
-	Dockermediatypes ImageMediaTypes = "DockerMediaTypes"
+	ImageMediaTypesDockermediatypes ImageMediaTypes = "DockerMediaTypes"
+
+	// Deprecated: use ImageMediaTypesDockermediatypes instead
+	Dockermediatypes ImageMediaTypes = ImageMediaTypesDockermediatypes
+
+	ImageMediaTypesOcimediatypes ImageMediaTypes = "OCIMediaTypes"
```

Comment:
Oh, I just noticed these values aren't in the right casing in the API:

```diff
"""Compression algorithm to use for image layers."""
enum ImageLayerCompression {
-  Gzip
-  Zstd
-  EStarGZ
-  Uncompressed
+  GZIP
+  ZSTD
+  E_STAR_GZ
+  UNCOMPRESSED
}

"""Mediatypes to use in published or exported image metadata."""
enum ImageMediaTypes {
-  OCIMediaTypes
-  DockerMediaTypes
+  OCI_MEDIA_TYPES
+  DOCKER_MEDIA_TYPES
}
```

I wonder if that was to preserve the capitalization in <code>EStar<strong>GZ</strong></code> and <code><strong>OCI</strong>MediaTypes</code>, but if it's producing `Estargz` and `Ocimediatypes` it's not working out. It would at least be `EStarGz` and `OciMediaTypes`, unless we pass it through the acronym thing, which is needed in a lot of places already.

Reply:
Yeah, but I think the opposite :) The SCREAMING_SNAKE_CASE is a convention in GraphQL, so I would expect this:

```graphql
"""Mediatypes to use in published or exported image metadata."""
enum ImageMediaTypes {
  OCI_MEDIA_TYPES @value("OCIMediaTypes")
  DOCKER_MEDIA_TYPES @value("DockerMediaTypes")
}
```